pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -1243,18 +1244,18 @@
       "StableVersions": [
         "4.0.0",
         "4.3.0",
-        "4.4.0",
```

Comment:
I just realized what you have matches master but we should have this version in master as well so please keep this version here.